Source/FolioReaderContainer.swift

```diff
@@ -188,4 +188,11 @@ public class FolioReaderContainer: UIViewController {
     override public func preferredStatusBarStyle() -> UIStatusBarStyle {
         return isNight(.LightContent, .Default)
     }
+
+	// MARK: - Public
+
+	public func readerCenter() -> FolioReaderCenter {
```

Comment:
Oh, didn't see this one. 
In general the new method would support an non singleton usage. But as there might be many places in the FolioReader which doesn't support a usage with multiple instances this doesn't matter at the moment.